ParisBNF/abb/BNFabb88.xml

```diff
@@ -0,0 +1,303 @@
+                                </q>
+                            </incipit>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf" quantity="182">182</measure>
+                                    <dimensions type="outer"/>
+                                    <height unit="mm" cert="medium">250</height>
+                                    <width unit="mm" cert="medium">200</width>
+                                    
+                                </extent>
+                                <foliation>Foliation in the upper right corner of the folia.</foliation>
+                                <collation>
+                                    <note>The quire structure cannot be ascertained on the basis of the available images.</note>
+                                </collation>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="18"> 
+                                    <ab type="pricking">Pricking is partly visible.</ab>
+                                    <ab type="ruling">Ruling is not visible.</ab>
+                                    <ab type="ruling" subtype="pattern" cert="low">1A-1A-1A1A/0-0/0-0/C</ab>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>  
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <seg type="script">18th-century script</seg> 
+                                <seg type="ink">Black, red.</seg>
+                                <desc>Careful, irregular; by a good scribe. <persName ref="PRS1071dAbbadi"/> describes the handwriting as <foreign xml:lang="fr">ʻguịlḥ moderneʼ</foreign>.
+                                </desc>
+                            </handNote>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="miniature">
+                                <locus target="#2v"/>
+                                <desc><ref type="authFile" corresp="AT1042HagiographicScene">: <persName ref="PRS9848Walatta"/></ref>Black and white image does not allow any certain interpretation of the miniature.</desc>
```

Comment:
This encoding
`<desc><ref type="authFile" corresp="AT1042HagiographicScene">: <persName ref="PRS9848Walatta"/></ref>Black and white image does not allow any certain interpretation of the miniature.</desc>`

makes no sense as you are creating a link inside of a link, and have ":" as the only textual content of the node, so your encoding would be read by the application as

![image](https://github.com/user-attachments/assets/a5e0a63c-64c0-4916-92d7-8b08b728b395)
(when there is a textual content of a node then only that one is printed so you only see ":"

If you remove ":" (`<ref type="authFile" corresp="AT1042HagiographicScene"><persName ref="PRS9848Walatta"/></ref>Black and white image does not allow any certain interpretation of the miniature.` then the outer link will be printed producing, for 

![image](https://github.com/user-attachments/assets/99b6dd42-d4aa-41c9-92de-f909f2fe2913)

If you want the name of the Art theme and the name of the saint, you have to either have both links separately, i.e. close ref `<ref type="authFile" corresp="AT1042HagiographicScene"/>` in order to print the link "[Hagiographic Scene](https://betamasaheft.eu/AT1042HagiographicScene)" before person and write `<ref type="authFile" corresp="AT1042HagiographicScene"/> <persName ref="PRS9848Walatta"/> Black and white image does not allow any certain interpretation of the miniature. `

or write the description inside of the ref, i.e. `<ref type="authFile" corresp="AT1042HagiographicScene">Walatta Petros doing something</ref>` as you nicely did below for d2

If you are not sure what you are seeing you do not have to assign an art theme or produce any other mark up